### DISCUSSION TITLE
Add Comments that link to other stories

### DISCUSSION THREAD

**Discussion by cjduncana** [storybookjs/storybook#15057] [Help]
I want to link to another story from the comments. I've been following [this resource](https://storybook.js.org/tutorials/design-systems-for-developers/react/en/document/) to document my components. Whenever I want to link to another page, I can add an anchor tag. I can't do the same with links in Storybook. I can't provide the absolute URL since it will only work in one environment but not another. If I provide a relative URL, it's relative to the iframe, so it doesn't link correctly. Any suggestions?

**Comment by yannbf** [comment]
What do you mean by comments? If you want to link via a story you can use this addon https://storybook.js.org/addons/@storybook/addon-links

**Reply by cjduncana** [reply]
Thanks for the prompt response 🚀 
Sadly, this addon only allows linking inside a component, not in its comments. You can head over to the **Extending Documentation** section of [this resource](https://storybook.js.org/tutorials/design-systems-for-developers/react/en/document/) to see how one can document components with JSDoc comments.

**Reply by M055277702** [reply]
FLEXList
